INTERNALS.md

```diff
@@ -0,0 +1,226 @@
+## Swabbie Background
+
+This doc presents an overview of Swabbie concepts and hopes to provide some background information on how the components work together. 
+It's a good place to start if you want to add a new resource or provider to Swabbie.
+
+### Taxonomy
+
+**Resource**: something that will be cleaned by Swabbie
+
+**Work Configuration**: The configuration for a distinct cloud provider + resource + regions + settings
+
+**Account**: a cloud provider's way of grouping resources
+
+**Namespace**: a [SwabbieNamespace](swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/model/SwabbieNamespace.kt) that represents a group of ResourceType in a cloud provider/account/region.
+
+### What does Swabbie look at?
+
+Swabbie was build to clean up things that Spinnaker knows about. 
+Some examples of those types of resources are Images, Snapshots, Autoscaling Groups, Load balancers, and Firewall Rules.
+Identifying that these resources exist is easy.
```

Comment:
```suggestion
Identifying that these resources exist is easy;```

Reply:
This didn't really come through... but I'd merge these two sentences with a semi-colon.